lib/application.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Application
+ * @constructor
+ * @param {Object} client Catapult client
+ */
+
+var Application = function (client) {
+	/**
+	 * List the user's applications
+	 * @param {Object} params Parameters for filtering applications
+	 * @param {Number} [params.page] The specified page requested when querying
```

Comment:
@rrao24 here is an example on how we do this in [messages](https://github.com/bandwidthcom/node-bandwidth/blob/2.0-preview/lib/message.js#L80)